lxd/instance/drivers/driver_qemu.go

```diff
@@ -6895,7 +6896,7 @@ func (d *qemu) devlxdEventSend(eventType string, eventMessage map[string]any) er
 func (d *qemu) Info() instance.Info {
 	data := instance.Info{
 		Name:     "qemu",
-		Features: []string{},
+		Features: make(map[string]any),
```

Comment:
What does the https://github.com/lxc/lxd/blob/master/lxd/instance/drivers/load.go#L176 look like in the logs now with this change? I'm wondering if we need to alter that line to just log the feature names, rather than their specific options.